### PR TITLE
added queue and stack implementation in java

### PR DESCRIPTION
**Pull request by ZohaibSoomro**

### CODE REVIEW

File: src/main/java/org/example/datastructures/queue/ArrayQueue.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package queue;
```
Comment (BimalRajGyawali) [inline src/main/java/org/example/datastructures/queue/ArrayQueue.java:6]: This should be 
```java
package org.example.datastructures.queue;
```

**Review by BimalRajGyawali — CHANGES_REQUESTED** [review]
Please use correct package path

File: src/main/java/org/example/datastructures/queue/ArrayQueue.java (hunk 1 above)
Comment (ZohaibSoomro) [reply src/main/java/org/example/datastructures/queue/ArrayQueue.java:6]: ok

**Comment by ZohaibSoomro** [comment]
done that

File: src/main/java/org/example/datastructures/queue/ArrayQueue.java (hunk 1 above)
Comment (ZohaibSoomro) [reply src/main/java/org/example/datastructures/queue/ArrayQueue.java]: you can merge it now, thanks for your cooperation.

**Review by BimalRajGyawali — APPROVED** [review]

**Pull request merged**